clj-refactor.el

```diff
@@ -3653,7 +3661,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-create-fn-from-e
         (buffer-substring (point-min) (point-max)))
     s))
 
-(defun cljr--is-keyword? (s)
+(defun cljr--sans-ns-keyword? (s)
```

Comment:
I am reluctant to do that as the new name expresses what the function does better than the original name. no misunderstandings no subtle bugs.